auth/app/build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation 'com.google.android.material:material:1.2.1'
 
-    implementation "com.google.firebase:firebase-auth-ktx:20.0.1"
+    implementation "com.google.firebase:firebase-auth-ktx:20.0.2"
```

Comment:
The build here is [failing](https://github.com/firebase/snippets-android/pull/273/checks?check_run_id=1790981672) with
```
org.jetbrains.kotlin.codegen.CompilationException: Back-end (JVM) Internal error: Couldn't inline method call 'userProfileChangeRequest' into updateProfile()
```
Downgrading to `auth-ktx:20.0.1` builds successfully.

@vkryachko was there any change made to auth-ktx recently? I don't see anything in the release notes.

Reply:
Also tracking this here:
https://youtrack.jetbrains.com/issue/KT-44375

Reply:
hm, we did not change anything, would be interesting to see what jetbrains reply

cc @rlazo